deployment/deployment.json

```diff
@@ -2001,6 +2001,35 @@
       }
     }
   },
+  "vesta-finance": {
+    "schema": "lending",
+    "base": "vesta-finance",
+    "protocol": "vesta-finance",
+    "deployments": {
+      "vesta-finance-arbitrum": {
+        "network": "arbitrum",
+        "status": "dev",
```

Comment:
you can change this to prod now since it passed QA

Reply:
Sorry, overlook it yesterday.

Reply:
np